django_cloud_deploy/tests/unit/cloudlib/billing_test.py

```diff
@@ -107,18 +148,34 @@ def test_enable_project_billing_fail(self):
             self._billing_client.enable_project_billing('invalid-project-id',
                                                         BILLING_ACCOUNT_NAME)
 
-    def test_list_billing_accounts(self):
+    def test_list_billing_accounts_no_page_token(self):
         accounts = self._billing_client.list_billing_accounts()
-        self.assertEqual(
-            len(BILLING_ACCOUNT_LIST_RESPONSE['billingAccounts']),
-            len(accounts))
+        self.assertEqual(BILLING_ACCOUNT_LIST_RESPONSE['billingAccounts'],
+                         accounts)
+
+    def test_list_billing_accounts_with_page_token(self):
+        responses = [
+            BILLING_ACCOUNT_LIST_RESPONSE,
+            BILLING_ACCOUNT_LIST_RESPONSE_NO_TOKEN
+        ]
+        cloudbillingservice_fake = CloudBillingServiceFake(
+            billing_account_responses=responses)
+        billing_client = billing.BillingClient(cloudbillingservice_fake)
+        accounts = billing_client.list_billing_accounts()
+        self.assertCountEqual(ALL_BILLING_ACCOUNTS, accounts)
```

Comment:
Shouldn't you check the contents too?

Reply:
Looks like "[assertCountEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual)" will check the contents.

I also tried this function with a simple example:

```python
    def test_assert_count_equal(self):
        d1 = {'a': 1}
        d2 = {'b': 2}

        d3 = {'c': 3}
        l1 = [d1, d2]
        l2 = [d2, d1]
        l3 = [d1, d3]

        self.assertCountEqual(l1, l2)
        self.assertCountEqual(l1, l3)
```

The first assertion will pass and the second will fail.

Reply:
My bad.